docs/02-server-installation.md

```diff
@@ -6,7 +6,7 @@ For installing docker, follow the Docker installation instructions found here: h
 Start the kvrocks service:
 
 ```shell
-$ docker run -it -p 6666:6666 kvrocks/kvrocks
+$ docker run -it -p 6666:6666 -v /var/lib/kvrocks:/var/lib/kvrocks kvrocks/kvrocks
```

Comment:
```suggestion
$ docker run -it -p 6666:6666 -v /var/lib/kvrocks:/var/lib/kvrocks apache/kvrocks
```

Also update image url and I think this may be an enhancement to Kvrocks - that is, we support docker run with some configure instead of modifying in place at Dockerfile.